src/utils/emailService.js

```diff
@@ -5,25 +5,24 @@ import dotenv from "dotenv";
 dotenv.config(); // Load environment variables
 
 
```

Comment:
Another good catch on all the formatting issues here. This is were a CI would come in handy so I don't forget to manually check them 🤦